index.html

```diff
@@ -18,40 +18,41 @@
   <header class="header">
     <div class="header__top">
       <div class="header__top-top container">
-        <img src="image/logo.png" alt="" class="header__logo">
-        <div class="header__search">
+        <a href="index.html">
+          <img src="image/logo.png" alt="logo_Analytics" class="header__logo">
+        </a>
+        <form class="header__search" action="/buscar" method="get">
           <input class="header__search-input" type="text" placeholder="¿Qué buscas?">
           <button class="header__search-button">
             <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-              <path fill-rule="evenodd" clip-rule="evenodd" d="M7 12C9.76142 12 12 9.76142 12 7C12 4.23858 9.76142 2 7 2C4.23858 2 2 4.23858 2 7C2 9.76142 4.23858 12 7 12Z" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-              <path d="M14.0001 14L10.5334 10.5333" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+              <desc>Logo de buscas?</desc>
+              <path fill-rule="evenodd" clip-rule="evenodd" d="M7 12C9.76142 12 12 9.76142 12 7C12 4.23858 9.76142 2 7 2C4.23858 2 2 4.23858 2 7C2 9.76142 4.23858 12 7 12Z" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+              <path d="M14.0001 14L10.5334 10.5333" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
             </svg>
           </button>
-        </div>
+        </form>
         <div class="header__location-hello">
           <div class="header__location">
-            <img class="header__location-img" src="image/location.png" alt="location">Todas
-              <a class="nav-link" href="#">Seleccionar tienda</a>
+            <img class="header__location-img" src="image/location.png" alt="logo location">Todas <a class="nav-link" href="#">Seleccionar tienda</a>
           </div>
           <div class="bord"></div>
-        
           <div class="header__hello">
-            <img class="header__hello-img" src="image/user.png" alt="hello"> Hola!
-              <a class="nav-link" href="#">Ingresa a tu cuenta</a>
+            <img class="header__hello-img" src="image/user.png" alt="logo user_login"> Hola! <a class="nav-link" href="#">Ingresa a tu cuenta</a>
```

Comment:
Otra vez escribiendo para maquinas? 😆 
Tambien parece ser solo decorativa
```suggestion
            <img class="header__hello-img" src="image/user.png" alt=""> Hola! <a class="nav-link" href="#">Ingresa a tu cuenta</a>
```